azure-functions/acbs-function/api.js

```diff
@@ -9,6 +9,9 @@
  * All the function have argument validation check and return object verification in
  * case err object does not have expected properties due to network connection, SSL verification or other issues.
  */
+const endpoint = process.env.MULESOFT_API_UKEF_TF_EA_URL;
```

Comment:
can we call it something like `acbsUrl` - or something like this. just to be more descriptive as to what the endpoint is doing